server/utils/ssr.tsx

```diff
@@ -92,12 +93,16 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 			<meta
 				key="t2"
 				property="og:title"
-				content={collection?.kind === 'book' ? collection.title : title}
+				content={!isPub ? collection?.title || title : title}
```

Comment:
Feel free to ignore these opinions, but I find the combination of the ternary and the short circuiting `||` in this expression to be a little taxing to read. `!isPub && collection?.title ? collection?.title : title` is a bit easier on my eyes personally. I also think this section would scan a bit better if it were extracted to a variable either way!

Reply:
thanks! Definitely see what you mean. Lmk what you think of the update I just pushed. If nothing else, I think the extra variable helps to understand the intent here much more.

Reply:
update looks great, much clearer!